index.html

```diff
@@ -109,13 +109,13 @@ <h3>Concept and Initial Report</h3>
     </div>
 
     <select multiple class="filter--scope">
-      <option value="Regional">Regional</option>
-      <option value="Citywide">Citywide</option>
-      <option value="City Center/Business District">
+      <option value="Regional" selected>Regional</option>
```

Comment:
There was no need to set `selected` in JavaScript; it can be set in HTML directly. That's more obvious.